osmaxx/core/templates/pages/about_us.html

```diff
@@ -0,0 +1,28 @@
+{% extends "osmaxx/base.html" %}
+{% load i18n %}
+
+{% block main %}
+<div class="container-row content">
+    <h2>About OSMaxx</h2>
+    <p>
+        OSMaxx (short for "OpenStreetMap arbitrary excerpt export") cuts out OpenStreetMap data, processes it to
+        geodata, converts it to typical GIS fileformats and offers those for download.
+    </p>
+    <h3>Useful links</h3>
+    <p>Can be found in our separate <a href="{% url 'pages:downloads' %}">downloads page</a></p>
+
+    <h3>Impressum</h3>
+    <p>
+        OSMaxx has been developed by the Geometalab at Hochschule Zürich.
+        Geometalab can be reached via email: <a href="mailto:geometalab@hsr.ch">geometalab@hsr.ch</a>.
+    </p><p>
+        <strong>Geometalab</strong><br />
+        HSR Hochschule für Technik Rapperswil<br />
+        Institute for Software<br />
+        c/o Stefan Keller<br />
+        Oberseestrasse 10<br />
+        Postfach 1475<br />
+        CH-8640 Rapperswil<br />
```

Comment:
> now shorter, better?

Actually, if we do keep the address (and IANAL, TINLA, but I guess the law says we must), let's use the postal address as it should be used on envelopes when sending snail mail to us. I guess that corresponds to the longer address as it was before 5b1e738d35be4671e89566f843737a70dac6c31e.

That is, unless HSR has some guidelines on what to use for impressums. @sfkeller do you know whether such guidelines exist?

> I went for `company information`

What if we call the page just "about" and this section "About Us"? That would go well with the "About OSMaxx" section above it.